board_report

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+command=$1
+OPENLAB_TOPDIR=/usr/local/openlab
+
+source $OPENLAB_TOPDIR/Include/common.sh
+source $OPENLAB_TOPDIR/Include/userop.sh
+source $OPENLAB_TOPDIR/Include/boardop.sh
+source $OPENLAB_TOPDIR/Include/telnetop.sh
+source $OPENLAB_TOPDIR/Include/bmcop.sh
+source $OPENLAB_TOPDIR/Include/board_serial_op.sh
+source $OPENLAB_TOPDIR/Include/board_power_op.sh
+source $OPENLAB_TOPDIR/Include/pduop.sh
+
+board_all_no=`grep -E "^(BOARD)" $OPENLAB_CONF_DIR/$BOARD_INFO_FILE | grep -Po "(?<=BOARD)([^:]*)" | tr '\n' " "`
+board_all_usr=$(cut -d ":" -f1 $OPENLAB_CONF_DIR/$USER_INFO_FILE | grep ^[a-zA-Z]|tr '\n' " ")
+
+################# Check the current usage of all the boards########################
+function busy()
+{
+    printf "%3s: %8s is used by -> %s\n" "Board No" "Board Type" "usr"
+    for value in ${board_all_no[*]}
+    do
+        usr=$(get_board_current_user $value)
+        board_type=`grep -E "^(BOARD${value}:).*" $OPENLAB_CONF_DIR/$BOARD_INFO_FILE | grep -Po "(?<=type=)([^ ,]*)"`
+        printf "%3s:  %8s is used by  -> %s\n"  "${value}" "${board_type}" "${usr}"
+    done
+}
+
+################## Check all the boards are owned by which user ###################
+function usr()
+{
+    echo "board_no is possessed as followsing list:"
+    printf "%3s: %8s -> Users list\n" "Board No" "Board Type"
+    for value in ${board_all_no[*]}
+    do
+        board_type=`grep -E "^(BOARD${value}:).*" $OPENLAB_CONF_DIR/$BOARD_INFO_FILE | grep -Po "(?<=type=)([^ ,]*)"`
+        usrinfo=`grep -E "boards=.*\b${value}\b" $OPENLAB_CONF_DIR/$USER_INFO_FILE| cut -d ":" -f1 | grep ^[a-zA-Z]|tr '\n' ";" | sed "s/;/; /g"`
+        if [ x"" = x"$usrinfo" ]; then
+            printf "%3s: is not exists\n" "${value}"
+        else
+            printf "%3s: %8s  -> %s\n"  "${value}" "${board_type}" "${usrinfo}"
+        fi
+    done
+}
+
+############## Check how many people are there in each organization ###############
+function org()
+{
+    a=`grep -Po "^([^#]*)" $OPENLAB_CONF_DIR/$USER_INFO_FILE | grep -Po "(?<=org=)[^,]+" | awk '{a[$0]++} END {for(i in a) printf("%-40s: appears ->%-4s \n"),i, a[i]}' | sort`
```

Comment:
What is the meaning of 'appears'? Please add a head line like "Area                    User count".